.flake8

```diff
@@ -14,4 +14,7 @@
 ignore = E261, E123, E221, E203, W503, W504, E226, E722, E126, E121, E731, E402
 per-file-ignores = interface_example.py:E501, tetris_bot.py:E501
 exclude = .git, __pycache__, opcodes.py, __init__.py
+per-file-ignores =
```

Comment:
This is a duplicate of line 15

Reply:
Damn, you're right. Then I think we should remove the first line, as it didn't work.